sdk/src/androidTest/java/com/mapbox/maps/TestUtils.kt

```diff
@@ -1,14 +1,15 @@
 package com.mapbox.maps
 
 import android.view.View
+import android.widget.FrameLayout
 import androidx.core.view.children
 import java.util.concurrent.CountDownLatch
 import java.util.concurrent.TimeUnit
 import java.util.concurrent.TimeoutException
 
-internal fun MapView.hasChildView(view: View) = children.any { it == view }
+internal fun FrameLayout.hasChildView(view: View) = children.any { it == view }
 
-internal fun MapView.getChildViewIndex(view: View) = children.indexOf(view)
+internal fun FrameLayout.getChildViewIndex(view: View) = children.indexOf(view)
```

Comment:
As discussed in the last kotlin group session, it's generally a bad practice to extend Android system APIs, but I guess it's less of a problem for internal methods, and in tests.